codelog.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.post_install_message = 'To start using the codelog run `codelog setup` and fill the `template.yml` file'
   spec.required_ruby_version = ">= 2.1.10"
 
-  spec.add_runtime_dependency "thor", "~> 0.19"
+  spec.add_runtime_dependency "thor", "~> 1.0"
```

Comment:
Duplicates #49 ..should not be a problem tho

Reply:
You know what, I must have built this branch based on that pull request rather than master